ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -210,14 +212,15 @@ void OnScreenKeyboard::drawPlaceHolderDisplayString() {
   displayStrWidth_=((textWidth+OSK_PLACEHOLDER_RESERVED_LENGTH) < oskLayout_.placeHolderLength ) ? (textWidth +OSK_PLACEHOLDER_RESERVED_LENGTH) :oskLayout_.placeHolderLength;
 
 /* Display Cursor*/
+  if(showCursor_){
   textWidth=0;
```

Comment:
Can you correct the alignment of this aswell

Reply:
Yes I did